.github/workflows/json-compability.yml

```diff
@@ -0,0 +1,23 @@
+name: Compability Check
+
+on:
+  pull_request:
+
+jobs:
+  compatible:
+    name: Is change incompatible
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - uses: actions/setup-node@v3
```

Comment:
there is also a v4 of setup-node

Reply:
not sure whether this solves my problems ;)